nilearn/datasets/func.py

```diff
@@ -1275,3 +1277,192 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
+
+    Technical details:
+    [2] Smith, S.M. et al. Nat. Neurosci. 18, 1565-1567 (2015).
+
+    [3] N.Filippini, et al. Distinct patterns of brain activity in young carriers
+    of the APOE-e4 allele.
+    Proc Natl Acad Sci USA (PNAS), 106:7209-7214, 2009.
+
+    [4] S.Smith, et al. Methods for network modelling from high quality rfMRI data.
+    Meeting of the Organization for Human Brain Mapping. 2014
+
+    [5] J.X. O'Reilly et al. Distinct and overlapping functional zones in the cerebellum
+    defined by resting state functional connectivity.
+    Cerebral Cortex, 2009.
+
+    Disclaimer
+    ----------
+    IMPORTANT: This is open access data. You must agree to Terms and conditions
+    of using this data before using it,
+    available at: http://humanconnectome.org/data/data-use-terms/open-access.html
+    Open Access Data (all imaging data and most of the behavioral data)
+    is available to those who register an account at ConnectomeDB and agree to
+    the Open Access Data Use Terms. This includes agreement to comply with
+    institutional rules and regulations. This means you may need the approval
+    of your IRB or Ethics Committee to use the data. The released HCP data are
+    not considered de-identified, since certain combinations of HCP Restricted
+    Data (available through a separate process) might allow identification of
+    individuals. Different national, state and local laws may apply and be
+    interpreted differently, so it is important that you consult with your IRB
+    or Ethics Committee before beginning your research. If needed and upon request,
+    the HCP will provide a certificate stating that you have accepted the
+    HCP Open Access Data Use Terms. Please note that everyone who works with
+    HCP open access data must review and agree to these terms, including those
+    who are accessing shared copies of this data. If you are sharing
+    HCP Open Access data, please advise your co-researchers that they must
+    register with ConnectomeDB and agree to these terms.
+    Register and sign the Open Access Data Use Terms at
+    ConnectomeDB: https://db.humanconnectome.org/
+    """
+    url = "http://www.nitrc.org/frs/download.php/8037/Megatrawls.tgz"
+    opts = {'uncompress': True}
+
+    # dataset terms
+    dimensionalities = [25, 50, 100, 200, 300]
+    timeseries_methods = ['multiple_spatial_regression', 'eigen_regression']
+    output_matrices = ['correlation', 'partial_correlation']
+
+    message = ("Invalid {0} name is given: {1}. "
+               "Please choose either of them {2}")
+
+    inputs = [dimensionality, timeseries, matrices]
+    standards = [dimensionalities, timeseries_methods, output_matrices]
+    error_names = ['dimensionality', 'timeseries', 'matrices']
+
+    for each_input, standard, name in izip(inputs, standards, error_names):
+        if not isinstance(each_input, list):
+            raise TypeError("Input given for {0} should be in list. "
```

Comment:
should be **a** list, maybe you can use `collections.Sequence` rather than list, i.e. you could argue tuple could work.